lib/subspace/commands/init.rb

```diff
@@ -0,0 +1,81 @@
+require 'fileutils'
+require 'erb'
+require 'subspace/commands/base'
+puts "Required init"
+class Subspace::Commands::Init < Subspace::Commands::Base
+  @@provision_templatedir = File.join(File.dirname(__FILE__), '..', 'template', 'provision')
+  @@dest_dir = "config/provision"
+
+  def initialize(args, options)
+    run
+  end
+
+  def run
+    FileUtils.mkdir_p File.join @@dest_dir, "group_vars"
+    FileUtils.mkdir_p File.join @@dest_dir, "host_vars"
+    FileUtils.mkdir_p File.join @@dest_dir, "vars"
+
+    copy ".gitignore"
+    template "ansible.cfg"
+    template "hosts"
+    template "group_vars/all"
+    create_vault_pass
+    environments.each do |env|
+      @env = env
+      @hostname = hostname(env)
+      template "group_vars/template", "group_vars/#{env}"
+      template "host_vars/template", "host_vars/#{env}"
+      create_vars_file_for_env env
+      template "playbook.yml", "#{env}.yml"
+    end
+
+    puts """
+    1. Create a server.
+
+    2. Set your server's location:
```

Comment:
None of these words are accurate anymore are they? Isn't this replaced with the ruby file?